darshan-util/pydarshan/tests/test_plot_dxt_heatmap.py

```diff
@@ -0,0 +1,461 @@
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+import darshan
+from darshan.experimental.plots import heatmap_handling
+from darshan.experimental.plots import plot_dxt_heatmap
```

Comment:
minor, for a follow-up: can do the imports from the same module without repeating the path